### PR TITLE
Update for new veto_files option

### PR DESCRIPTION
**Pull request by alphatangoalpha**
**Description:**
Updating documentation to reflect version 9.0 of the samba addon changes.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [X ] Branch: current
- [X ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_addons/samba.markdown [hunk 1]
```diff
@@ -55,4 +62,9 @@ allow_hosts:
   required: false
   default: '`["10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]`'
   type: list
+veto_files:
+  description: List of files that are neither visible nor accessible. Useful to stop clients from littering the share with temporary hidden files (ex: macOS .DS_Store, Windows Thumbs.db)
```
Comment (klaasnicolaas) [inline source/_addons/samba.markdown]:
```suggestion
  description: List of files that are neither visible nor accessible. Useful to stop clients from littering the share with temporary hidden files (i.e., macOS .DS_Store, Windows Thumbs.db)
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_addons/samba.markdown (hunk 1 above)
Comment (alphatangoalpha) [reply source/_addons/samba.markdown]: ex: or e.g. is grammatically correct here. i.e. should not be used.

Comment (klaasnicolaas) [reply source/_addons/samba.markdown]: Then change it to `e.g.` 😉

**Comment by frenck** [comment]
A small and friendly notification.

Are you planning on moving forward on this PR? 
I didn't see any activity recently, while there are still things to do. I hope you consider picking it up soon! 👍

File: source/_addons/samba.markdown (hunk 1 above)
Comment (alphatangoalpha) [reply source/_addons/samba.markdown]: Changed to e.g. per request

**Review by frenck — APPROVED** [review]
Thanks, @alphatangoalpha! 👍

**Pull request merged**